templates/dashboard/taxes/zero_page.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'dashboard/base_zero_page.html' %}
+{% load i18n %}
+
+{% block title %}{% trans "No taxes found." %}{% endblock %}
+{% block content %}
+  {% blocktrans %}
+    Taxes define tax rates used in European Union countries and are fetched automatically.<br>
```

Comment:
This is misleading, taxes are not fetched automatically, one needs to fetch them via commandline